src/main/java/com/rtb/projectmanagementtool/user/AuthOps.java

```diff
@@ -0,0 +1,46 @@
+package com.rtb.projectmanagementtool.user;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import java.util.*;
+
+public final class AuthOps {
+
+  private DatastoreService datastore;
+  public String cookieName;
+  public String[] cookieValues;
+  public Cookie userCookie;
+
+  public AuthOps(DatastoreService datastore) {
+    this.datastore = datastore;
+    cookieName = "loginStatus";
+    cookieValues = {"out", "null"};
+  }
+
+  public void loginUser() {
+    String AuthID;
+    userCookie = new Cookie(cookieName, cookieValues);
+    
```

Comment:
Try to follow the steps you have in the Detailed Design.  Step #1.a is saying "using cookies, c heck loggedin status", which means we'll first need to look at existing cookies to see if we already have loggedInUser tag in there.